scripts/docker/Dockerfile

```diff
@@ -27,8 +27,12 @@ RUN apt-get update && \
 	apt-get autoremove -y && \
 	apt-get clean && \
 	find /var/lib/apt/lists/ -type f -not -name lock -delete; \
-# add user
-	useradd -m -u 1000 -U -s /bin/sh -d /polkadot polkadot
+# add user and link ~/.local/share/polkadot to /data
+	useradd -m -u 1000 -U -s /bin/sh -d /polkadot polkadot && \
+	mkdir -p /polkadot/.local/share && \
+	mkdir /data && \
+	chown -R polkadot:polkadot /data && \
```

Comment:
for `-R` it would allow `/data` to exist before and contain data. `mkdir /data` doesn't - it will fail when the folder already exists. maybe
```
mkdir -p /data /polkadot/.local/share
```